server/routes/availability.js

```diff
@@ -0,0 +1,22 @@
+const express = require("express");
+const router = express.Router();
+const { protect } = require("../middleware/auth");
+const {
+  createSchedule,
+  editSchedule,
+  getActiveSchedule,
+  getAvailability,
+  setActiveSchedule,
+} = require("../controllers/availability");
+
+router.route("/createSchedule").post(protect, createSchedule);
+
+router.route("/:scheduleId").put(protect, editSchedule);
```

Comment:
Could we add some express validation middleware here to ensure the schedule id is always passed in? There is example in the server codebase to follow if you need.